composites/Plugin/ContentAnalysis/reducers/tests/keywordResultsReducerTest.js

```diff
@@ -0,0 +1,394 @@
+import { keywordResultsReducer } from "../contentAnalysis/keywordResultsReducer";
+import { UPDATE_SEO_RESULT, SET_SEO_RESULTS, REMOVE_KEYWORD } from "../../actions/contentAnalysis";
+
+// The SET_SEO_RESULTS action
```

Comment:
Just to be curious, if we get remote data, do we assume there are no duplicate result ids?